python/pyspark/sql/readwriter.py

```diff
@@ -220,9 +220,22 @@ def options(self, **options: "OptionalPrimitiveType") -> "DataFrameReader":
 
         Examples
         --------
+        Set a single option using the `option` method:
+
         >>> spark.read.option("key", "value")
```

Comment:
Should actually remove this example because it's docstring for `options`, and we should also fix streaming's reader/wrtier too at https://github.com/apache/spark/pull/43358.

I will leave it to you if you update this PR (then I close mine) or close :-).